package com.trybe.calcularidade.exception;

/**
 * Classe DataFuturaException.
 **/
public class DataFuturaException extends RuntimeException {

  public DataFuturaException(String message) {
    super(message);
  }

}
